web/beacon-app/src/components/layout/DashLayout.tsx

```diff
@@ -7,6 +14,28 @@ type DashLayoutProps = {
 };
 
 const DashLayout: React.FC<DashLayoutProps> = ({ children }) => {
+  // const Store = useOrgStore.getState() as any;
+  const { tenants, wasTenantsFetched } = useFetchTenants();
+  const hasTenants =
+    tenants?.tenants && Array.isArray(tenants?.tenants) && tenants?.tenants?.length > 0;
+  // const tenantID = tenants?.tenants[0]?.id;
+
+  // ensure the tenant is loaded correctly otherwise we need to throw an error
+  useEffect(() => {
+    if (wasTenantsFetched) {
+      invariant(
+        hasTenants,
+        t`Tenant is not loaded correctly. Please contact us at support@rotational.io for assistance.`
```

Comment:
Is it possible that a user may not know what a tenant is if they see this message? Would a more generic message like, `Something went wrong. Please contact us at support@rotational.io for assistance.` be less confusing to users while also giving them an action to take.

Reply:
On the error page, it's generally helpful for users to understand what information to convey when attempting to contact the admin. This might include providing an error code or any relevant details that can assist us in pinpointing the source of the issue.
But I will update the content with your suggestion for now.